scripts/src/mock_client.ts

```diff
@@ -411,18 +421,13 @@ async function outdatedJWT() {
 	const appClient = new SampleAppClient();
 
 	let jwt = await appClient.getRegisterJWT(userId, moment().add(1, "days").unix());
-	try {
-		await MarketplaceClient.create({ jwt });
-		throw new Error("shouldn't be able to register with JWT with iat in the future");
```

Comment:
throwing the error here is useless because you catch it the next line... This is fixed in `expectToThrow`